web/content/docs/benchmarks/hydro-mechanics/InjectionProduction.md

```diff
@@ -0,0 +1,73 @@
+---
```

Comment:
This file need to have a so called frontmatter at the beginning. See [other](https://raw.githubusercontent.com/ufz/ogs/master/web/content/docs/benchmarks/elliptic/groundwater-flow-robin.md) for reference.

Reply:
I added a formatter manually,  please take a look.